src/Simulation/Simulators/QuantumProcessor/QuantumProcessorDispatcher.cs

```diff
@@ -27,19 +26,19 @@ public IQuantumProcessor QuantumProcessor
         /// <summary>
         /// 
         /// </summary>
-        /// <param name="quantumProcessor">An instance of a class implementing <see cref="IQuantumProcessor"/> interface to be wrapped. If the parameter is null <see cref="QuantumProcessorBase"/> is used.</param>
-        /// <param name="qubitManager">An instance of a class implementing <see cref="IQubitManager"/> interface. If the parameter is null <see cref="QubitManagerTrackingScope"/> is used.</param>
+        /// <param name="quantumProcessor">An instance of a class implementing <see cref="IQuantumProcessor"/> interface to be wrapped.</param>
+        /// <param name="qubitManager">An instance of a class implementing <see cref="IQubitManager"/> interface. If the parameter is null <see cref="QubitManager"/> is used.</param>
         /// <param name="randomSeed">A seed to be used by Q# <a href="https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.random">Microsoft.Quantum.Intrinsic.Random</a> operation.</param>
-        public QuantumProcessorDispatcher(IQuantumProcessor? quantumProcessor = null, IQubitManager? qubitManager = null, int? randomSeed = null)
+        public QuantumProcessorDispatcher(IQuantumProcessor quantumProcessor, IQubitManager? qubitManager = null, int? randomSeed = null)
```

Comment:
This may have release note impact and warrant a roll-forward of the minor version number, since it's a breaking change to the interface for this class, and could break ) third-party simulators not tracked in our cross-repo build.

Reply:
There are plenty of breaking changes in this PR (like removing an entire class), so the idea is to accumulate these breaking changes across the runtime repo and then get them over with. The goal is to walk out with an API we are ready to commit to maintaining.